src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -114,7 +116,28 @@ public void setUpdateDate(Timestamp updateDate) {
     this.updateDate = updateDate;
   }
 
-  /**
+  public Integer getCountCollaborators() {
+    ArrayList<String> group = new ArrayList<>();
+    if (Objects.nonNull(getUserId())) {
+      group.add(getUserId().toString());
+    }
+
+    List<Collaborator> collabs = data.getInternalCollaborators();
+
+    if (Objects.nonNull(collabs)) {
+      collabs.addAll(data.getLabCollaborators());
+
+      for (Collaborator c: collabs) {
+        String userID = c.getUuid();
+        if (!group.contains(userID)) {
+          group.add(userID);
+        }
```

Comment:
This will likely not do what it seems to be doing. The UUID is only used for the UI. There could be two `j.smith@someplace.edu` collaborators with different UUIDs, so this would consider the to be two different people. What you should probably be looking at is a lower-cased email address instead. Even this isn't ideal since a single person can be referenced by multiple emails by different researchers, but it's the best thing we have with collaborators.